docs/jenkins_gerrit_integration.md

```diff
@@ -200,9 +200,16 @@ CodeChecker log -b "build command" -o compile_cmd.json
 # Run the tests. If this fails, the result will be verified -1.
 make tests
 
+export CC_CHANGED_FILES="$WORKSPACE/files-changed"
+export CC_SKIPFILE="$WORKSPACE/skipfile"
+
+# gerrit_changed_files_to_skipfile.py script can be found in the bin directory in the CodeChecker package.
+python gerrit_changed_files_to_skipfile.py $CC_CHANGED_FILES $CC_SKIPFILE
```

Comment:
Do not we need to use python3 binary here? For example in the Makefiles we are using python3 binary.

Reply:
What if the jenkins environment only has python2?